cmd/endorsements/main.go

```diff
@@ -110,16 +110,19 @@ func runListQueriesCommand(
 func runCommand(config *common.Config, command string, args []string, logger *zap.Logger) error {
 	var err error
 
+	quiet := true
 	if config.Debug {
 		if logger, err = zap.NewDevelopment(); err != nil {
 			return err
 		}
 		defer logger.Sync() //nolint
+
+		quiet = false
 	}
 
 	em := endorsement.NewManager()
 	if err := em.InitializeStore(
-		config.PluginLocations, config.EndorsementStoreName, config.EndorsementStoreParams,
+		config.PluginLocations, config.EndorsementStoreName, config.EndorsementStoreParams, quiet,
```

Comment:
this may be a `config` parameter too, I think?

Reply:
I considered that, but this really more of internal decision. This flags doesn't really set the logging level, but suppresses output that would be there otherwise. The only reason to do that is for testing (if this flag is false, the logging is set to WARN level).